third_party/docstringchecker/lint.py

```diff
@@ -0,0 +1,490 @@
+# Copyright (c) 2013 The Chromium OS Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
```

Comment:
To be clear, this is third-party code we're pulling in. Ideally we wouldn't have to copy it locally, but there isn't a good way of getting it otherwise. You don't need to code review it, though.